Onboarding Portal (#270)
golemfactory/golem-docs: src/pages/docs/creators/javascript/guides/switching-to-mainnet.md

{% /alert %}

> #### Acquiring MATIC/ETH and GLM tokens directly via Golem Onboarding Portal

Reviewer: ```suggestion
#### Acquiring MATIC/ETH and GLM tokens directly via the Golem Onboarding Portal
```

Author: OK

1. Visit our [Onboarding Portal](https://glm.golem.network/)
2. Click **Get Started** or **Get GLM** button
3. Follow the instructions provided by the Portal

> #### Acquiring MATIC/ETH and GLM tokens from YAGNA context via Golem Onboarding Portal

Reviewer: ```suggestion
#### Acquiring MATIC/ETH and GLM tokens from the YAGNA context via the Golem Onboarding Portal
```

Author: OK

> 1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):

Reviewer: ```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

> 1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):

Reviewer: ```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

> 1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):

Reviewer: ```suggestion
1. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):
```

> 1a. For Polygon network, enter following command into your Terminal window (Yagna has to be running in the background):

Reviewer: ```suggestion
1. For the Polygon network, enter the following command into your Terminal window (Yagna has to be running in the background):
```

Author: OK

```bash
yagna payment fund --network=polygon
```

> 1b. Alternatively for Ethereum network, enter following command:

Reviewer: ```suggestion
Alternatively for the Ethereum network, enter the following command:
```

Author: OK

```bash
yagna payment fund --network=mainnet
```

> 2. Follow instructions presented into newly opened Web Page of your default browser

Reviewer: ```suggestion
2. Follow the instructions presented on the newly opened Web Page of your default browser
```

Author: OK
